doc/related_projects.rst

```diff
@@ -12,6 +12,12 @@ Interoperability and framework enhancements
 These tools adapt scikit-learn for use with other technologies or otherwise
 enhance the functionality of scikit-learn's estimators.
 
+- `Machine Learning <https://github.com/jeff1evesque/machine-learning>`_ an automated build
```

Comment:
I further reduced the language to be more concise. However, I'm not sure if I'm able to rename the repository at this time.  But, I could adjust the link text, to something you suggest?